codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/UnionGenerator.java

```diff
@@ -51,48 +52,41 @@ public void run() {
             writer.write("is$L()", symbol.getName());
         }).write("");
 
-        // Create type aliases for each member that satisfy the interface.
+        // Create structs for each member that satisfy the interface.
         for (MemberShape member : shape.getAllMembers().values()) {
             Symbol memberSymbol = symbolProvider.toSymbol(member);
             String exportedMemberName = symbol.getName() + symbolProvider.toMemberName(member);
```

Comment:
What do you think about adding the quantifier `Member` between the Union shape name and the member name?

e.g. `FooMemberBar`

Reply:
Sure